CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). If you introduce breaking changes, please group them together in the "Changed" section using the **BREAKING:** prefix.
 
+## [v0.1.3] - 2023-?
+
+### Added
+
+- Authentication status now persisted in VS Code (#94)
```

Comment:
Can we make the issue/pr numbers links to make it easier for folks to get back to the relevant URL?

Reply:
Done. Thanks